core/src/main/java/com/homeaway/streamingplatform/StreamRegistryApplication.java

```diff
@@ -48,10 +48,7 @@
 import org.rocksdb.BlockBasedTableConfig;
 import org.rocksdb.Options;
 
-import com.homeaway.streamingplatform.configuration.InfraManagerConfig;
-import com.homeaway.streamingplatform.configuration.StreamRegistryConfiguration;
-import com.homeaway.streamingplatform.configuration.StreamValidatorConfig;
-import com.homeaway.streamingplatform.configuration.TopicsConfig;
+import com.homeaway.streamingplatform.configuration.*;
```

Comment:
we need to put coding standards into this project.

* **no wildcard imports** except for unit-test helper/matchers please.